apps/trading/e2e/tests/spot_market/test_spot.py

```diff
@@ -0,0 +1,222 @@
+    fields = [
+        ["24 Hour Volume", "1.00(108 USDT)"],
+        ["Open Interest", "-"],
+        ["Best Bid Volume", "99"],
+        ["Best Offer Volume", "99"],
+        ["Best Static Bid Volume", "1"],
+        ["Best Static Offer Volume", "1"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_key_details(page: Page, vega: VegaServiceNull):
+    page.get_by_test_id(market_title_test_id).get_by_text(
+        "Key details").click()
+    market_id = vega.find_market_id("Bitcoin / Tether USD (Spot)")
+    short_market_id = market_id[:6] + "…" + market_id[-4:]
+    fields = [
+        ["Market ID", short_market_id],
+        ["Name", "Bitcoin / Tether USD (Spot)"],
+        ["Status", "Active"],
+        ["Trading Mode", "Continuous"],
+        ["Market Decimal Places", "1"],
+        ["Position Decimal Places", "2"],
+        ["Settlement Asset Decimal Places", "5"], # need to be remove after resolve this one  #6183
+        [" Tick Size", "1"]
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_instrument(page: Page):
+    page.get_by_test_id(market_title_test_id).get_by_text("Instrument").click()
+    fields = [
+        ["Market Name", "Bitcoin / Tether USD (Spot)"],
+        ["Code", "BTC/USDT-SPOT"],
+        ["Product Type", "Spot"],
+        ["Quote Name", "USDT"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_base_asset(page: Page, vega: VegaServiceNull):
+    page.get_by_test_id(market_title_test_id).get_by_text(
+        "Base asset").click()
+    tdai_id = vega.find_asset_id("BTC")
+    tdai_id_short = tdai_id[:6] + "…" + tdai_id[-4:]
+    fields = [
+        ["ID", tdai_id_short],
+        ["Type", "Builtin asset"],
+        ["Name", "BTC"],
+        ["Symbol", "BTC"],
+        ["Decimals", "5"],
+        ["Quantum", "0.00001"],
+        ["Status", "Enabled"],
+        ["Max faucet amount", "10,000,000,000.00"],
+        ["Infrastructure fee account balance", "0.003"],
+        ["Global reward pool account balance", "0.00"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_quote_data(page: Page, vega: VegaServiceNull):
+    page.get_by_test_id(market_title_test_id).get_by_text(
+        "Quote asset").click()
+    tdai_id = vega.find_asset_id("USDT")
+    tdai_id_short = tdai_id[:6] + "…" + tdai_id[-4:]
+    fields = [
+        ["ID", tdai_id_short],
+        ["Type", "Builtin asset"],
+        ["Name", "USDT"],
+        ["Symbol", "USDT"],
+        ["Decimals", "5"],
+        ["Quantum", "0.00001"],
+        ["Status", "Enabled"],
+        ["Max faucet amount", "10,000,000,000.00"],
+        ["Infrastructure fee account balance", "0.003"],
+        ["Global reward pool account balance", "0.00"],
+    ]
+    validate_info_section(page, fields)
+
+def test_market_spot_info_metadata(page: Page):
+    page.get_by_test_id(market_title_test_id).get_by_text("Metadata").click()
```

Comment:
Should there be any asserts here?

Reply:
it is empty so validation is by clicking